apps/provider/src/app/admin/setup/page.tsx

```diff
@@ -1,12 +1,22 @@
+import type { Metadata } from 'next'
 import { redirect } from "next/navigation";
 import { Stepper } from "./stepper";
 import { auth } from "@/auth";
 import {UserRole} from "@/db/schema";
-import { GetApplicationSettings } from "@/actions/ApplicationSettings";
+import { GetApplicationSettings, GetAppName } from "@/actions/ApplicationSettings";
 import React from "react";
 
 export const dynamic = "force-dynamic";
 
+export async function generateMetadata(): Promise<Metadata> {
+  const appName = await GetAppName()
+
+  return {
+    title: `Setup - ${appName}`,
+    description: "Light up your earnings with Igniter",
```

Comment:
Maybe we want to use the app name here instead of igniter?

```suggestion
    description: `Light up your earnings with ${appName}`,
```